site/docs/configuration.md

```diff
@@ -25,6 +25,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.metadata.compression-codec   | none               | Metadata compression codec; none or gzip           |
 | write.metadata.metrics.default     | truncate(16)       | Default metrics mode for all columns in the table; none, counts, truncate(length), or full |
 | write.metadata.metrics.column.col1 | (not set)          | Metrics mode for column 'col1' to allow per-column tuning; none, counts, truncate(length), or full |
+| write.target-file-size             | Long.MAX_VALUE     | Controls the size of files generated to target about this many bytes. |
```

Comment:
Would it make sense to rename the prop to `write.target-file-size-bytes` to be consistent with other write props?

Reply:
At the same time, we have `read.split.target-size` and `read.split.open-file-cost`, so I am ok with both options.

Reply:
Agreed that being more specific can only help. Will change to `write.target-file-size-bytes`.

Reply:
Fixed.